Configuration/SwissBitcoinPaySettings.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Smartstore.Core.Configuration;
+
+namespace SmartStore.SwissBitcoinPay.Settings
+{
+    public class SwissBitcoinPaySettings : ISettings
+    {
+        /// <summary>
+        /// The url of the Swiss Bitcoin Pay API
+        /// </summary>
+        public string ApiUrl { get; set; }

```

Comment:
The parameter does not have to be configurable nor visible to users. Is it possible to remove it?

Reply:
OK, done.
This parameter is hidden in the config page.